server/controllers/Profiles.js

```diff
@@ -68,6 +68,34 @@ class Profiles {
       return serverError(res);
     }
   }
+
+  /**
+   * @name view
+   * @description allows a user to view other users profile
+   * @param {object} req request object
+   * @param {object} res response object
+   * @returns {json} the json response been return by the server
+   * @memberof ProfilesController
+   */
+  static async view(req, res) {
+    try {
+      const {
+        params: { id }
+      } = req;
+      const userProfile = await User.findById(id);
+      if (!userProfile) {
+        return serverResponse(res, 404, {
+          error: 'user does not exist'
+        });
+      }
+      delete userProfile.dataValues.password;
+      return serverResponse(res, 200, {
```

Comment:
can we create userResponse helper thats basically works like serverResponse, but have an added responsibility of deleting the password from the user project before sending to client?